devel/libhttpd/Portfile

```diff
@@ -1,32 +1,37 @@
-PortSystem 1.0
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
```

Comment:
if you add a modeline you need to make sure you conform to it (i.e., no tabs, indent uniformly with spaces as multiples of four). The `lint` warnings in the CI tell you this and you should do `port lint` locally as stated in the PR checklist. I'll repeat it here again, the checklist is not optional - at least do the lint and build it locally before submitting a PR.

Reply:
Building locally is always done before submission of anything, of course. I will correct other things now.